src/controls/util.ts

```diff
@@ -85,10 +83,9 @@ export function findCornerQuadrant(
   control: Control
 ) {
   //  angle is relative to canvas plane
-  const angle = fabricObject.getTotalAngle(),
-    cornerAngle =
-      angle + radiansToDegrees(Math.atan2(control.y, control.x)) + 360;
-  return Math.round((cornerAngle % 360) / 45);
+  const rotation = calcPlaneRotation(fabricObject.calcTransformMatrix()),
+    cornerAngle = rotation + Math.atan2(control.y, control.x) + twoMathPi;
+  return Math.round((cornerAngle % twoMathPi) / quarterPI);
```

Comment:
can we type this function saying it returns a number explicitly?
I was confused for a memoment thinking we moved from degrees to radians